app/src/firebase/InitializeDatabase.ts

```diff
@@ -4,7 +4,8 @@ import { getAuth, connectAuthEmulator } from "firebase/auth";
 import { getFunctions, connectFunctionsEmulator } from "firebase/functions";
 
 const firebaseConfig = {
-    apiKey: process.env.REACT_APP_API_KEY,
+    // exception is thrown if api key is undefined even if using emulator
+    apiKey: process.env.REACT_APP_API_KEY || "dummy-api-key",
```

Comment:
Could we move this to `app/package.json` instead? Similar to how we set `REACT_APP_PROJET_ID`?
It may also be worth moving them to a `dotenv` file.

Reply:
Have set it up with a `.env.development` file which gets picked up appropriately